src/features/home/components/headers/HighlightThematicHomeHeader.tsx

```diff
@@ -24,40 +24,65 @@ export const HighlightThematicHomeHeader: FunctionComponent<HighligthThematicHea
   imageUrl,
   beginningDate,
   endingDate,
+  introductionTitle,
+  introductionParagraph,
 }) => {
   const { navigate } = useNavigation<UseNavigationType>()
   const onGoBack = useCallback(() => navigate(...homeNavConfig), [navigate])
 
   const dateRange = computeDateRangeDisplay(beginningDate, endingDate)
 
+  const shouldShowIntroduction = !!introductionTitle && !!introductionParagraph
+
   return (
-    <ImageBackground source={{ uri: imageUrl }}>
-      <StatusBar barStyle="light-content" animated />
-      <Spacer.TopScreen />
-      <View>
-        <BackButtonContainer>
-          <BackButton onGoBack={onGoBack} />
-        </BackButtonContainer>
-        <DateRangeCaptionContainer>
-          <DateRangeCaption>{dateRange}</DateRangeCaption>
-        </DateRangeCaptionContainer>
-      </View>
-      <TextContainer>
-        <ThematicHighlightGradient />
-        <BlackBackground>
-          {!!subtitle && (
-            <React.Fragment>
-              <Subtitle numberOfLines={1}>{subtitle}</Subtitle>
-              <Spacer.Column numberOfSpaces={1} />
-            </React.Fragment>
-          )}
-          <Title numberOfLines={2}>{title}</Title>
-        </BlackBackground>
-      </TextContainer>
-    </ImageBackground>
+    <React.Fragment>
+      <ImageBackground source={{ uri: imageUrl }}>
+        <StatusBar barStyle="light-content" animated />
+        <Spacer.TopScreen />
+        <View>
+          <BackButtonContainer>
+            <BackButton onGoBack={onGoBack} />
+          </BackButtonContainer>
+          <DateRangeCaptionContainer>
+            <DateRangeCaption>{dateRange}</DateRangeCaption>
+          </DateRangeCaptionContainer>
+        </View>
+        <TextContainer>
+          <ThematicHighlightGradient />
+          <BlackBackground>
+            {!!subtitle && (
+              <React.Fragment>
+                <Subtitle numberOfLines={1}>{subtitle}</Subtitle>
+                <Spacer.Column numberOfSpaces={1} />
+              </React.Fragment>
+            )}
+            <Title numberOfLines={2}>{title}</Title>
+          </BlackBackground>
+        </TextContainer>
+      </ImageBackground>
+      {shouldShowIntroduction ? (
+        <Introduction title={introductionTitle} paragraph={introductionParagraph} />
+      ) : null}
+    </React.Fragment>
   )
 }
 
+const Introduction: React.FC<{
+  title: string
+  paragraph: string
+}> = ({ title, paragraph }) => (
```

Comment:
```suggestion
type IntroductionProps = {
  title: string
  paragraph: string
}

const Introduction = ({ title, paragraph }: IntroductionProps) => (
```

Vous écrivez encore des `React.FC` vous ?